arkane/explorer.py

```diff
@@ -193,95 +194,103 @@ def execute(self, outputFile, plot, format='pdf', print_summary=True, speciesLis
             incomplete = False
             for T in Tlist:
                 for P in Plist:
-                    kchar = 0.0 #compute the characteristic rate coefficient by summing all rate coefficients from the reactant channel
-                    for rxn in self.network.netReactions:#reaction_model.core.reactions+reaction_model.edge.reactions:
-                        if set(rxn.reactants) == set(self.source) and len(rxn.products) == 1 and rxn.products[0].molecule[0].getFormula() == form:
-                            kchar += rxn.kinetics.getRateCoefficient(T=T,P=P)
-                        elif set(rxn.products) == set(self.source) and len(rxn.reactants) == 1 and rxn.reactants[0].molecule[0].getFormula() == form:
-                            kchar += rxn.generateReverseRateCoefficient(network_kinetics=True).getRateCoefficient(T=T,P=P)
```

Comment:
I think this chunk was added in a commit which is part of this PR:
`use the sum of the rate coefficients for all net rates from the`, https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1545/commits/f236722653d56b7628ac8aca340c22b18ce73f0e

Can you merge these instead of deleting the code?

Reply:
I think this is fixed too.